src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/MemberInfoExtensions.cs

```diff
@@ -61,6 +61,19 @@ public static bool IsNonNullableReferenceType(this MemberInfo memberInfo)
 
         public static bool IsDictionaryValueNonNullable(this MemberInfo memberInfo)
         {
+#if NET6_0_OR_GREATER
+            var context = new NullabilityInfoContext();
+            var nullableInfo = memberInfo.MemberType == MemberTypes.Field
+                ? context.Create((FieldInfo)memberInfo)
+                : context.Create((PropertyInfo)memberInfo);
+
+            if (nullableInfo.GenericTypeArguments.Length != 2)
+            {
+                throw new InvalidOperationException("Expected Dictionary to have two generic type arguments.");
```

Comment:
Could you update the message with a bit more detail, such as the type and the member names? Then if it happened, it would be much easier to debug from the message alone.